Zend/zend_exceptions.stub.php

```diff
@@ -32,26 +34,20 @@ public function __construct(string $message = UNKNOWN, int $code = 0, ?Throwable
 
     public function __wakeup() {}
 
-    /** @return string */
-    final public function getMessage() {}
+    final public function getMessage(): string {}
 
     /** @return int */
     final public function getCode() {}
```

Comment:
I had to leave this return type alone because some DB-related tests failed due to returning a string...

Reply:
Yes, this is a known issue :(